packages/mdc-drawer/README.md

```diff
@@ -437,55 +437,83 @@ document.body.addEventListener('MDCDrawer:closed', () => {
 });
 ```
 
-## `MDCDrawer` Properties and Methods
+<!-- docgen-tsdoc-replacer:start -->
```

Comment:
Let's make this obvious by mentioning that this is autogenerated and do not edit.


```html
<!-- docgen-tsdoc-replacer:start **DO NOT EDIT, This section is automatically generated** -->
```